weatherMan.py

```diff
@@ -0,0 +1,175 @@
+from datetime import datetime
+import os
+import glob
+import sys
+
+#months dictionary
+months = {
+        '1': "January",
+        '2': "February",
+        '3': "March",
+        '4': "April",
+        '5': "May",
+        '6': "June",
+        '7': "July",
+        '8': "August",
+        '9': "September",
+        '10': "October",
+        '11': "November",
+        '12': "December"
+    }
+
+
+#GETTING AND DISPLAYING YEARLY DATA
+def getYearData():
+    if(sys.argv[2]=='dubai'):
+        directory='Dubai_weather'
+    elif(sys.argv[2]=='lahore'):
+        directory='lahore_weather'
+    elif(sys.argv[2]=='murree'):
+        directory='Murree_weather'

```

Comment:
these condition are repeated in both yearly and monthly calculations. Generalize it.